docs/user-guide/src/ironic/ironic_installation.md

```diff
@@ -62,6 +62,10 @@ prevent eavesdropping.
 
 ### Installing with Kustomize
 
+In the quickstart guide, we have demonstrated [how to install ironic with kustomize](../quick-start.md#deploy-ironic), by creating an ironic kustomization overlay.
```

Comment:
Wrap lines at 80, even though it is not forced here due legacy. Makes it easier for me to enable it later.

Reply:
Thanks. Does it look ok now?

Reply:
Much better, thank you!